checkout-extension/shopify.extension.toml.liquid

```diff
@@ -1,35 +1,51 @@
 # Learn more about configuring your checkout UI extension:
 # https://shopify.dev/api/checkout-extensions/checkout/configuration
 
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
 api_version = "2023-07"
 
 [[extensions]]
-name = "{{ name }}"
 type = "ui_extension"
+name = "{{ name }}"
```

Comment:
I find these config files look nicest with `type` as the first line, since it is generally he shortest and the only one of these three related fields that does not contain developer-specific content.